src/crypto.ts

```diff
@@ -15,6 +18,8 @@ import {
   decodeUTF8,
 } from 'tweetnacl-util'
 
+import { decode as decodeBase64ToUint8Array } from '@stablelib/base64'
```

Comment:
is it possible to use `decodeBase64` from `tweetnacl-util` instead? I think there have been problems with decoding non-English characters using `stablelib` in the past - @liangyuanruo and @karrui can correct me if I'm wrong

Reply:
Yeah we had to revert that in a previous PR, I think there was a backwards compatibility issues https://github.com/opengovsg/formsg-javascript-sdk/commit/c9b16ecdbf259074461de515de81aa9842c8a371